packages/dashboard/src/components/lists/List.js

```diff
@@ -19,100 +19,91 @@ import {
   CardBody,
 } from 'shards-react'
 
-
-
-
-const List = ({
-  accounts,
-  user,
-  historicQuotes,
-  globalSelectedAccount,
-}) => {
-  
-
-  
+const List = ({ items, listTitle }) => {
+  const renderListItems = () => {
+    return items.map(item => (
+      <tr key={item.id}>
+        <td>
+          {item.stock.symbol} <sub>{item.stock.name}</sub>
+        </td>
+        <td>{'$ ' + item.quote.amount}</td>
+        <td>
+          <div
+            style={{
+              width: '60%',
+              padding: '8px',
+              borderRadius: '6px',
+              background:
+                item.quote.previous_close < item.quote.amount
+                  ? 'lightgreen'
```

Comment:
using `lightgreen` and I presume the red rgba. I wouldnt mix rgba and variable defaults. The greens look a bit different from what we use in the smallStats, maybe use the same green + red there for consistency?

Reply:
yup good call. I was planning on playing with the colours, I'm also a bit colourly challenged when it comes to red/green.

Reply:
LOL #colourblind